### DISCUSSION TITLE
Coordinate problems

### DISCUSSION THREAD

**Discussion by octohhy** [simnibs/simnibs#350] [9. Results and analysis]
When using the Probe plugin, how do I display the coordinate points I set in the graph. I mean I don't know now where this coordinates are in the graph.
![brain](https://github.com/simnibs/simnibs/assets/112316569/f348f075-c138-435f-9140-da6329677c35)

**Comment by TorgeW** [comment]
![image](https://github.com/simnibs/simnibs/assets/122801447/29f989bf-0448-467f-bfd7-2feedfde45bd)

Gmsh creates a point with the probed value at the probed position. 
You have to set the view settings as shown in the screenshot to better see the location.
